src/status_im/profile/screen.cljs

```diff
@@ -155,10 +163,13 @@
 
     [view st/profile-property-with-top-spacing
      [selectable-field {:label     (label :t/phone-number)
+
                         :editable? false
                         :value     (if (and phone (not (str/blank? phone)))
                                      (format-phone-number phone)
-                                     (label :t/not-specified))}]
+                                     (label :t/not-specified))
+                        ;; TODO: should this be changed?
+                        :on-press  navigate-to-phone-change}]
```

Comment:
The profile view you are talking about is another user's profile. You can access it by opening a chat with some user and pressing "Profile" in popup menu after that.
So the answer is no, we don't need to navigate to console chat in this case.